<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MotorWindow</class>
 <widget class="QWidget" name="MotorWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pb_Enable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Disable">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Disable</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Stop">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sp_Steps">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="l_Steps">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>62</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Steps</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_Rotate">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_Continuous">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuous</string>
   </property>
  </widget>
  <widget class="QDial" name="dial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="maximum">
    <number>359</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="wrapping">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_StepDone">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>50</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Step Done</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_Steps_Remais">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Step(s) remains: XXX</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_IsEnable">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="cb_Tick">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>290</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
